src/graphql/candyMachine.ts

```diff
@@ -203,29 +199,61 @@ const runUploadV2UsingHiddenSettings = async (
     hash: new Array(32).fill(0),
   };
 
-  const pubkeyFields = ['solTreasuryAccount', 'collection'];
+  // these fields come in as strings, but we need to transform them to pubkey objects
+  const pubkeyFields = [
+    'solTreasuryAccount',
```

Comment:
would prefer if this was an enum or at least consts since you refer to them in several places

Reply:
yeah, I want the API to be fully typed, but needs more clean up than just this